language-design.md

```diff
@@ -28,7 +103,54 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+Guy Steele gives us several traits of a good language in "Growing a Language".
+According to Steele, the size of the language is one important indicator. As he
```

Comment:
As a follow up to this, how would you determine the size of a language. Like what is too small and what's too large?
I agree with your answer that a small and growable language is preferred, but what are some identifiers of languages that are just the right size? You mention some features that a growable language would have, but I think you should also mention how one would judge the size of a language.